styles2.css

```diff
@@ -0,0 +1,528 @@
+
+.button {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.primary-button {
+  width: 6rem;
+  height: auto;
+  background-color: var(--primary-color);
+  color: #fff;
+  text-align: center;
+  border-radius: 0.5rem;
+  padding: 0.05rem;
+  border: 1px solid var(--primary-color);
+  cursor: pointer;
+}
+
+.secondary-button {
+  height: auto;
+  background-color: #fff;
+  color: var(--primary-color);
+  text-align: center;
+  border-radius: 0.5rem;
+  padding: 0.05rem;
+  width: 6rem;
+  border: 1px solid var(--primary-color);
+  cursor: pointer;
+}
+
+.floating-button {
+  width: 3rem;
+  height: 3rem;
+  border-radius: 50%;
+  background-color: var(--primary-color);
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  color: #fff;
+}
+
+.plus-button {
+  align-self: center;
+}
+
+/* Cards */
+
+.card {
+  margin-right: 1rem;
+  margin-bottom: 1rem;
+}
+
+.cards {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  flex-wrap: wrap;
+}
+
+.text-only-cards {
+  width: 15rem;
+  background-color: #e5e7eb;
+  height: 20rem;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+}
+
+.card-title {
+  align-self: center;
+}
+
+.vertical-card {
+  width: 15rem;
+  background-color: #e5e7eb;
+  height: 20rem;
+}
+
+.horizontal-card {
+  width: 20rem;
+  background-color: #e5e7eb;
+  height: 15rem;
+}
+
+/* Images */
+
+.container-image {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  flex-wrap: wrap;
+}
+
+.responsive-image {
+  width: 16rem;
+  height: 20rem;
+  margin-bottom: 1rem;
+}
+
+.round {
+  width: 16rem;
+  height: 16rem;
+  border-radius: 50%;
+}
+
+/* Lists */
+
+.lists-container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+/* Stacked Lists */
+
+.list-group {
+  width: 100%;
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.list-item {
+  width: 40%;
+  border: 1px solid var(--grey-color);
+  padding: 0.5rem;
+}
+
+/* Spaced Lists */
+
+.spaced {
+  margin-bottom: 1rem;
+}
+
+/* Modals */
+
+.modal-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.modal-box {
+  width: 20rem;
+  border: 1px solid var(--grey-color);
+  display: grid;
+  grid-template-rows: 1fr 1fr 1fr;
+  padding: 0rem;
+}
+
+.modal-title {
+  grid-row: 1/2;
+  padding: 0.05rem;
+  text-align: center;
+  border-bottom: 1px solid var(--grey-color);
+}
+
+.modal-text {
+  grid-row: 2/3;
+  padding: 2rem;
+  border-bottom: 1px solid var(--grey-color);
+}
+
+.modal-buttons {
+  grid-row: 3/4;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+/* Text Utilities */
+
+.text-types {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.colored-normal-text {
+  color: var(--grey-color);
+}
+
+/* Rating */
+
+/* footer */
+
+.footer-ele {
```

Comment:
I have a suggestion you can update this class name to more descriptive one.

Reply:
Yes, I will make the necessary changes. Thank You :)